Modules/Core/GPUCommon/include/itkGPUImageOps.h

```diff
@@ -38,14 +38,15 @@ class GPUImageOps
   GPUImageOps() = default;
   virtual ~GPUImageOps() = default;
 
+  GPUImageOps() = delete;
+  virtual ~GPUImageOps() = delete;
```

Comment:
This can not bee defined as both "default" and "delete"   Why was the delete added here?

Reply:
Mistake when moving from private to public. Should be resolved now.